posts/zzz_DO_NOT_EDIT_roxy.shinyl.../roxy.shinylive_-_shinylive_applications_in_roxygen_documentation.qmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "roxy.shinylive - shinylive applications in roxygen documentation"
+author:
+  - name: Pawel Rucki
+description: "Introducing a `roxy.shinylive` - a new tool for Shiny package developers"
+# Note that the date below will be auto-updated when the post is merged.
+date: "2024-10-07"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Shiny, Technical]
+# Feel free to change the image
+image: "roxy.shinylive.gif"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_roxy.shinyl..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+![](roxy.shinylive.gif)
+
+Continuing my exploration of WebR, I'm happy to introduce a new tool for Shiny package developers - [`roxy.shinylive`](https://insightsengineering.github.io/roxy.shinylive/). The package is designed for anyone building their Shiny applications or modules as a R package. With just a few lines of code, you can easily embed an `iframe` to a Shinylive application based on the code from the "Examples" section of your documentation.
```

Comment:
Might be nice to link your previous post here?

Reply:
ok - added!